packages/lib-user/src/components/GroupAllProjects/GroupAllProjects.js

```diff
@@ -0,0 +1,94 @@
+import { arrayOf, bool, shape, string } from 'prop-types'
+import { useTranslation } from '../../translations/i18n.js'
+
+import { usePanoptesProjects, useStats } from '@hooks'
+import { AllProjects, ContentBox, HeaderLink, Layout } from '@components/shared'
+
+const STATS_ENDPOINT = '/classifications/user_groups'
+
+// props are passed from GroupContainer via cloneElement
+function GroupAllProjects({ authUser, group }) {
```

Comment:
Something to consider with this component is that it's wrapped by GroupContainer, which is necessary to handle join notifications and fetch the group's data. GroupAllProjects is similar to Contributors in that way. However, Contributors also handles `adminMode` and `membership` props for further checking of permissions to view individual contributors.

I don't think GroupAllProjects needs the same permissions check because it can just follow whether a group is public or not. If a group is public, allow anyone to see its `/projects` subpath. If a group is private, the "You don't have permission to view this group" message displays. Please let me know if you disagree with any of that behavior!

Reply:
Makes sense to me 👍